marc_to_solr/lib/traject_config.rb

```diff
@@ -709,9 +709,16 @@
 
 to_field 'language_facet', marc_languages
 
-to_field 'publication_place_facet', extract_marc('008[15-17]') do |_record, accumulator|
+to_field 'publication_place_facet', extract_marc('008[15-17]') do |record, accumulator, context|
   places = accumulator.map { |c| Traject::TranslationMap.new('marc_countries')[c.strip] }
   accumulator.replace(places.compact)
+rescue ArgumentError => exception
+  logger.error "#{record['001']} Encountered and Arugument Error exception: #{exception}"
+  raise exception if exception.message != "invalid byte sequence in UTF-8"
```

Comment:
@carolyncole thank you, it looks great. Can you please fix the typo and then we merge it? '..an Argument..'